lib/tasks/data/statistics.rake

```diff
@@ -16,75 +16,80 @@ namespace :check do
       ActiveRecord::Base.logger = nil
 
       errors = []
-      team_ids = Team.joins(:project_medias).where(project_medias: { user: BotUser.smooch_user }).distinct.pluck(:id)
+      team_ids = Team.joins(:project_medias).where(project_medias: { user: BotUser.pluck(:id) }).distinct.pluck(:id)
       current_time = DateTime.now
       puts "[#{Time.now}] Detected #{team_ids.length} teams with tipline data"
       team_ids.each_with_index do |team_id, index|
-        tipline_bot = TeamBotInstallation.where(team_id: team_id, user: BotUser.smooch_user).last
-        if tipline_bot.nil?
-          puts "[#{Time.now}] No tipline bot installed for team #{team_id}; skipping team"
-          next
-        end
+        TeamBotInstallation.where(team_id: team_id, user: BotUser.pluck(:id)).find_each do |bot|
+          tipline_message_statistics = Check::TiplineMessageStatistics.new(team_id)
+          date = ProjectMedia.where(team_id: team_id, user: bot.user).order('created_at ASC').first&.created_at&.beginning_of_day
+
+          next if date.nil?
+
+          team = Team.find(team_id)
+          languages = team.get_languages.to_a
+          if bot.user == BotUser.smooch_user
+            platforms = bot.smooch_enabled_integrations.keys 
+          else
+            platforms = Bot::Smooch::SUPPORTED_INTEGRATION_NAMES.keys
```

Comment:
**Question:**

There are two options here, use all `Bot::Smooch::SUPPORTED_INTEGRATION_NAMES.keys` or just use a dash (`-`). Which one would be preferred?

Reply:
I think this way is better!